docs/quickstart.md

```diff
@@ -0,0 +1,10 @@
+
+## Quickstart Guides
+
```

Comment:
I'd include a short introductory sentence here to help the reader understand these links. Here's my suggestion:
```suggestion
Killercoda provide an interactive environment for exploring KubeVirt scenarios:
```